lib/config/presets/index.ts

```diff
@@ -293,11 +294,15 @@ export async function resolveConfigPresets(
             error.validationError = `Preset is invalid (${preset})`;
           } else if (err.message === PRESET_PROHIBITED_SUBPRESET) {
             error.validationError = `Sub-presets cannot be combined with a custom path (${preset})`;
+          } else if (err.message === PRESET_INVALID_JSON) {
+            error.validationError = `Preset is invalid json (${preset})`;
```

Comment:
```suggestion
            error.validationError = `Preset is invalid JSON (${preset})`;
```

JSON is an abbreviation, so should be capitalized. 😉